matrix/plugins/glitch/utils.py

```diff
@@ -0,0 +1,17 @@
+from juju.client.client import ClientFacade
+import subprocess
+import yaml
+
+async def is_leader(model, rule, unit_name):
```

Comment:
Passing in rule because I was doing debug logging. We might want to think of a better way to pass the log around.